cartoframes/context.py

```diff
@@ -715,6 +715,13 @@ def query(self, query, table_name=None, decode_geom=False):
             Pandas data types are inferred from PostgreSQL data types.
             In the case of PostgreSQL date types, dates are attempted to be
             converted, but on failure a data type 'object' is used.
+
+        Examples:
+            Query a table in CARTO and write a new table that is result of query
```

Comment:
How about a bit more of a description about what's happening? Maybe something like 'Get 10 highest values from a table. This will write to a dataframe and create a table in CARTO.'?